processor/src/main/java/org/mapstruct/ap/internal/model/StreamMappingMethod.java

```diff
@@ -63,36 +65,39 @@ protected StreamMappingMethod instantiateMappingMethod(Method method, Collection
                 sourceParameterType.isIterableType() ) {
                 helperImports.add( ctx.getTypeFactory().getType( StreamSupport.class ) );
             }
-
+            Map<String, List<LifecycleCallbackMethodReference>> mappingReferencesMap = new HashMap<>(2);
+            mappingReferencesMap.put( "before", beforeMappingMethods );
+            mappingReferencesMap.put( "after", afterMappingMethods );
```

Comment:
Why are we changing this? Can we please keep the PR to only have the changes we need? If the problem was the fact that the constructor had too many arguments then let's turn Checkstyle on and off like we do in the constructor for the `BeanMappingMethod`